test/source/test.ts

```diff
@@ -55,6 +57,9 @@ ava.before('set config and mock api', async t => {
     const mockApi = await mock(line => mockApiLogs.push(line));
     closeMockApi = mockApi.close;
   }
+  const pubkey = pubkey2864E326A5BE488A;
+  const contact = await ContactStore.obj({ email: 'sams50sams50sept@gmail.com', name: 'Mr Sam', pubkey, client: 'pgp' });
+  await ContactStore.save(undefined, contact);
```

Comment:
Is it possible to put these lines somewhere closer to where they are being used?